inc/zoo/swar/SWAR.h

```diff
@@ -70,6 +79,18 @@ struct SWAR {
         // "don't care" bits when non-power-of-two bit lane sizes are supported
         LowerBits = MostSignificantBit - LeastSignificantBit;
 
+    template <typename Arg, std::size_t N, typename = std::enable_if_t<N == Lanes, int>>
+    constexpr
+    SWAR(Literals_t<NBits, T>, const Arg (&values)[N]) : m_v{0} {
+        auto result = T{0};
+        for (const auto arg : values) {
+            result = (result << NBits) | arg;
+        }
+        m_v = result;
+    }
+
+    constexpr static T MaxUnsignedLaneValue = ~(((~T{0}) << (NBits - 1)) << 1);
```

Comment:
I agree this constant is worth including, but not here, and not with this calculation.  See there are a bunch of constants in `SWAR`, you may, for example, put it at the end of the list like this:
```c++
MaxUnsignedLaneValue = LeastSignificantLaneMask;
```